File: 201909-1/cpp/main.cpp
#include <cstdio>

using namespace std;

int num1, num2;
int appleTotal = 0;
int maxIndex = 0;
int maxNum = 0;

int main()
{
    scanf("%d %d", &num1, &num2);

    for (int i = 0; i < num1; i++)
    {
        int appleNum;
        scanf("%d", &appleNum);

        int delCount = 0;
        for (int m = 0; m < num2; m++)
        {
            int delNum;
            scanf("%d", &delNum);
            delCount += delNum;
        }
        appleTotal += appleNum + delCount;
        if (delCount < maxNum)
        {
            maxIndex = i;
            maxNum = delCount;
        }
    }

    printf("%d %d %d\n", appleTotal, maxIndex + 1, -maxNum);

    return 0;
}

File: 201909-1/python/main.py
def main():
    num1, num2 = map(int, input().split())
    appleTotal = 0
    maxIndex = 0
    maxNum = 0
    for i in range(0, num1):
        arrs = [int(arr_str) for arr_str in input().split()]
        delNum = 0
        for m in range(0, num2):
            delNum += arrs[m + 1]
        appleTotal += arrs[0] + delNum
        if delNum < maxNum:
            maxIndex = i
            maxNum = delNum

    print(appleTotal, maxIndex + 1, -maxNum)


if __name__ == '__main__':
    main()
